test/core/sequencer/benchmark.js

```diff
@@ -29,20 +43,19 @@ test('benchmark all modules', function(t) {
     var end = Date.now();
     console.log('Module ' + mods[0] + ' ran in: ' + (end - global.start) + ' milliseconds');
     mods.splice(0, 1);
-    if (mods.length > 1) { //Last one is test module, we need not benchmark it
+    if (mods.length > 1) { // Last one is test module, we need not benchmark it
       sequencer.steps[global.idx].output.src = image;
       global.idx++;
       if (mods[0] === 'import-image' || (!!sequencer.modulesInfo(mods[0]).requires && sequencer.modulesInfo(mods[0]).requires.includes('webgl'))) {
```

Comment:
Here we can additionally exempt `fisheye-gl`. Although it might be more efficient to compare a list of exempted modules to a list on line https://github.com/publiclab/image-sequencer/pull/1093/files#diff-57fdbe0dfda9b326e4947de8d679f8b6R32

But why isn't this working? `sequencer.modulesInfo(mods[0]).requires.includes('webgl'))` ? That should be sufficient to exempt fisheye-gl:

https://github.com/publiclab/image-sequencer/blob/c3af98ea930f747e1055cfc1404e233b6a2c907c/src/modules/FisheyeGl/info.json#L4

Reply:
Well this  issue is not of benchmarks, its the module tests causing error.
We should skip that one and it would be ok!!

Reply:
So should I skip the fisheye and webgl modules here by deleting the tests for them!!

Reply:
Hmm, it seems you're right, so we can check if this is happening in tests elsewhere and is unrelated.

Reply:
So, then we should modify the test runner line to ensure that our combined benchmarks actually do run.

Reply:
Yes, this fisheyegl error is in all tests, not this PR only: https://travis-ci.org/publiclab/image-sequencer/jobs/547201993#L925-L955

Ug. #1113 -- which please also if you can try working on, that would be great - this is a perfect example of how it can cause trouble! ❤️

Reply:
Ya sure!! I will look into it very soon.